cms/methodology/models.py

```diff
@@ -80,6 +81,7 @@ class MethodologyPage(GenericTaxonomyMixin, BasePage):  # type: ignore[django-ma
         ),
         FieldPanel("content", icon="list-ul"),
         InlinePanel("related_pages", label="Related publications"),
+        *BundledPageMixin.panels,
```

Comment:
note: on `StatisticalArticlePage` we show this first. Let's do the same everywhere - we do want to be consistent, and in this case prioritise the information